api/src/main/java/graphql/nadel/instrumentation/ChainedNadelInstrumentation.java

```diff
@@ -55,62 +53,44 @@ public InstrumentationState createState(NadelInstrumentationCreateStateParameter
     @Override
     public InstrumentationContext<ExecutionResult> beginQueryExecution(NadelInstrumentationQueryExecutionParameters parameters) {
         return new ChainedInstrumentationContext<>(instrumentations.stream()
-                .map(instrumentation -> {
-                    InstrumentationState state = getStateFor(instrumentation, parameters.getInstrumentationState());
-                    return instrumentation.beginQueryExecution(parameters.withNewState(state));
-                })
-                .collect(toList()));
+            .map(instrumentation -> {
```

Comment:
Reformat, this should be the last time since there's actually a code style stored in the repository now.